exe/Main.cpp

```diff
@@ -61,9 +61,7 @@ int main(int argc, char** argv)
         exit(0);
     }
 
-    sofa::helper::logging::MessageDispatcher::addHandler(&sofa::helper::logging::MainLoggingMessageHandler::getInstance());
     sofa::helper::logging::MessageDispatcher::addHandler(&sofa::helper::logging::MainPerComponentLoggingMessageHandler::getInstance()) ;
-    sofa::helper::logging::MainLoggingMessageHandler::getInstance().activate();
```

Comment:
Does that mean that runSofaGLFW+SofaGLFW wont have the "per-component" message anymore ?
I suppose we should put the removed line in init() of SofaGLFW (like your did for Imgui)